src/app/game-state/character.service.ts

```diff
@@ -147,20 +147,7 @@ export class CharacterService {
     }
   }
 
-
-  condenseSoulCore(){
-    if (this.characterState.aptitudeGainDivider <= 10){
-      // double check we're not going over the max rank
-      return;
-    }
-    this.logService.addLogMessage(
-      "Your spirituality coelesces around the core of your soul, strengthening it and reforging it into something stronger.",
-      'STANDARD', 'STORY');
-    this.logService.addLogMessage(
-      "You now gain additional aptitude each time you reincarnate.",
-      'STANDARD', 'STORY');
-    this.characterState.condenseSoulCoreCost *= 10;
-    this.characterState.aptitudeGainDivider -= 10;
+  ascend(){
```

Comment:
Nitpick on the name of the function here. Maybe resetAttributes()?

Reply:
Or resetAptitudes()?

Reply:
Added. It's a better naming standard.